### PR TITLE
Configuration docs and releases on CI

### PR DESCRIPTION
**Pull request by joodie**

### CODE REVIEW

File: authorization-register/README.md [hunk 1]
```diff
@@ -25,12 +25,95 @@ policies and a PolicyStore protocol for adding and deleting policies.
...
+hard to test in isolation; most components depend on multiple services
+which in a production environment are usually hosted at third parties.
+
+Having to coordinate with third parties in order to correctly
+configure required services slows down development, testing &
+demonstrations. It can also make it difficult to create automated
+tests for edge cases (when the test case depends on very specific
+responses from services).
+
+BDI Services often rely on an Authorization Register. The BDI AR is a
```
[the remark this answers is not in the document]
Comment (joodie) [reply authorization-register/README.md:41]: Maybe we should talk about how this project should be branded.
